### PR TITLE
[MC-#14] - Create Text Input Component 

### LINKED ISSUE #14: [Feature] - Create Text Input Component
**Description**
I want a text input box to be created that will be used as the default for the whole site, following the prototype design. 
There are two types of text entry boxes:
- Normal input
- Outline input

**Additional context**
image

**Acceptance Criteria**
- Separate CSS files
- The code must be responsive
- Build successfully
- lint successfully
- Use color in variables
- Implement both types of inputs

### PR DESCRIPTION
**Pull request by RodrigoSoaresNascimento**
# Feature/Issue Description
 This PR creates a text box according to the design in the figma website.

# Changes Summary
- adding a new input text in home page.
- adding a new stye in css.style. 
- create a file  SCSS call input-box in the files scss

# How Has This Been Tested?
put the system to rotate and compress the screen and expand . The tests performed were to compress the screen diagonally, during compression the text box did not break, the following browsers were used, Google Chrome, Microsoft Edge, and Opera.

# Images (Opcional)
1 compress-image
image
2-normal-browser
image

### CODE REVIEW

File: src/assets/scss/style.scss [hunk 1]
```diff
@@ -18,3 +18,40 @@ body {
...
+    margin-bottom: 20px;
+}
+.input-box::placeholder{
+    color: $other-dark;
+}
+.input-box:focus{
+    background-color: $primary-light;
+    border: 2px solid $accent-light;
+}
+
```
Comment (tvps20) [inline src/assets/scss/style.scss]: All the code must be in a specific css file for this input

File: src/pages/components/index.html [hunk 2]
```diff
@@ -4,7 +4,22 @@ <h1 class="title">Components</h1>
...
+                </h3>
+                <p>Normal</p>
+                
+                <div class="forms-input mb-3">
+                    <input class="input-box" type="text" placeholder="First name">
+                
+                </div>
+            </div>
+               
+
```
Comment (tvps20) [inline src/pages/components/index.html]: Need to have a pattern, lots of spaces needed

**Review by tvps20 — CHANGES_REQUESTED** [review]
**Checkpoints:**
- Commit is not in default [MC-Task Number Message];
- PR name is not in the pattern [[MC-Task Number] - Story Title];
- Missing to fill in the pull request info;
- The team was not added to validate the pr;
- The pull request must be signed for you;
- A label must be added for this pull request;
- PR link not available in the group;

File: src/assets/scss/style.scss (hunk 1 above)
Comment (RodrigoSoaresNascimento) [reply src/assets/scss/style.scss]: done!

File: src/pages/components/index.html (hunk 2 above)
Comment (RodrigoSoaresNascimento) [reply src/pages/components/index.html]: done!

**Comment by feelsantiago** [comment]
image

Add min-width to input

File: src/pages/components/index.html [hunk 3]
```diff
@@ -4,7 +4,16 @@ <h1 class="title">Components</h1>
     </div>
 
     <div class="row text input">
-        <!-- Text input here -->
+        <div class="text-input-box">
+            <h3>Inputs</h3>
+
+            <p>Normal</p>
+
+            <div class="forms-input mb-3">
```
Comment (feelsantiago) [inline src/pages/components/index.html]: Where are these classes defined? They are necessary?

File: src/assets/scss/input-box.scss [hunk 4]
```diff
@@ -0,0 +1,35 @@
+.text-input-box{
```
Comment (feelsantiago) [inline src/assets/scss/input-box.scss]: The component sccs should have only the scss for the input and not for the purple box. The purple box should be place on components/style.scss

**Review by feelsantiago — CHANGES_REQUESTED** [review]
Good work! Minor fix to resolve only.

File: src/pages/components/index.html (hunk 3 above)
Comment (RodrigoSoaresNascimento) [reply src/pages/components/index.html]: I made this class according to the example entries on the bootstrap site, but they are not used in sccs modification so I removed it.

File: src/assets/scss/input-box.scss (hunk 4 above)
Comment (RodrigoSoaresNascimento) [reply src/assets/scss/input-box.scss]: done!

File: src/assets/scss/input-box.scss [hunk 5]
```diff
@@ -0,0 +1,43 @@
+.text-input-box {
+    width: 600px;
```
Comment (feelsantiago) [inline src/assets/scss/input-box.scss]: You should not define a fixed length for the input. The input must grow or shrink based on the container that is put in.

You should define a min-width for the input.

Comment (feelsantiago) [inline src/assets/scss/input-box.scss]: You should not define a fixed length for the input. The input must grow or shrink based on the container that is put in.

You should define a min-width for the input.

**Review by feelsantiago — CHANGES_REQUESTED** [review]

**Review by tvps20 — DISMISSED** [review]

File: src/assets/scss/input-box.scss (hunk 5 above)
Comment (RodrigoSoaresNascimento) [reply src/assets/scss/input-box.scss]: Done! added a minimum width of 200px and the size configured according to the container

**Review by feelsantiago — APPROVED** [review]
Great work!

File: src/assets/scss/input-box.scss (hunk 5 above)
Comment (RodrigoSoaresNascimento) [reply src/assets/scss/input-box.scss]: Done!

**Review by tvps20 — APPROVED** [review]

**Pull request merged**